action/protocol/staking/handlers.go

```diff
@@ -815,7 +820,7 @@ func fetchCaller(ctx context.Context, csm CandidateStateManager, amount *big.Int
 	}
 	gasFee := big.NewInt(0).Mul(actionCtx.GasPrice, big.NewInt(0).SetUint64(actionCtx.IntrinsicGas))
 	// check caller's balance
-	if gasFee.Add(amount, gasFee).Cmp(caller.Balance) == 1 {
+	if !caller.HasSufficientBalance(new(big.Int).Add(amount, gasFee)) {
```

Comment:
`gasFee.Add(amount, gasFee)` should work, like current code

Reply:
I like my code